Header/Header.module.less

```diff
@@ -46,6 +46,16 @@
 			text-indent: @agate-header-title-text-indent;  // Fix for characters like `j` being left-truncated because of being too close to the edge
 		}
 
+		.subtitle {
+			color: @agate-header-subtitle-text-color;
+			opacity: @agate-header-subtitle-opacity;
+			padding-top: @agate-header-subtitle-padding;
+		}
+
+		.titleAbove {
+			padding-bottom: @agate-header-subtitle-padding;
```

Comment:
I think this should be broken out into its own variable, specifically for `titleabove`.